### PR TITLE
only update the records with uuids in the file provided

### PR DESCRIPTION
**Pull request by craigrose**
Also moved all the related files into one directory.  This is a one use only thing.

### CODE REVIEW

File: scripts/backlog-1105/update-by-uuid.sh [hunk 1]
```diff
@@ -0,0 +1,40 @@
+#! /bin/bash
+
+# We are given a xml file and extract the geonetwork uuids from it to find records for updating
```
Comment (pmbohm) [inline scripts/backlog-1105/update-by-uuid.sh]: Why not a generic script we can use again?
The file input should be a list of UUID's produced by other means, such as https://github.com/aodn/utilities/blob/master/geonetwork/gn-tool/get-uuids-from-file.py

Comment (pmbohm) [reply scripts/backlog-1105/update-by-uuid.sh]: It could even be done in mafia java land?

Comment (craigrose) [reply scripts/backlog-1105/update-by-uuid.sh]: The uuid list and transform xsl files are now parameters

File: scripts/backlog-1105/update-by-uuid.sh [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+action="$1";
+
+for uuid in $(cat "$2")
+  do
+    case "$action" in
+      show)
+        # Display the records to be changed
+        java -jar ../../mafia/target/mafia-1.0.0.jar \
+            -url jdbc:postgresql://"$3"/geonetwork_aodn \
```
Comment (pmbohm) [inline scripts/backlog-1105/update-by-uuid.sh]: add `geonetwork_aodn` to $3 and your on a winner

Comment (craigrose) [reply scripts/backlog-1105/update-by-uuid.sh]: ok done that

**Pull request merged**